### ISSUE TITLE
Site-icon's focus ring is too tall, and is clipped beneath the site icon below it

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#14802]
## Description 
Focus ring is too tall, and is clipped beneath the site icon below it

## Steps to Reproduce

   1.  switch to the `Favorites` view in Top Sites via either the context-menu item or via `Customize` -> `Top Sites` -> `Favorites`
   2.  add > 6 favorites (up to 12)
   3.  start pressing `Tab` on the first row, and notice the focus ring

## Actual result:

The focus ring extends beneath the next site's favicon, and looks awkward due to being clipped

<img width="1117" alt="Screen Shot 2021-03-17 at 7 29 06 PM" src="https://user-images.githubusercontent.com/387249/111564682-661ccc80-8757-11eb-9a15-fe1ed19e1b32.png">

## Expected result:

No clipping into the next site's favicon; in Figma, the focus ring looks to merely comprise the favicon/tile, not the accompanying site text

<img width="568" alt="Screen Shot 2021-03-17 at 7 37 29 PM" src="https://user-images.githubusercontent.com/387249/111565252-489c3280-8758-11eb-835e-12579182f7f0.png">

## Reproduces how often: 
100%

## Brave version (brave://version info)

Brave | 1.24.2 Chromium: 89.0.4389.90 (Official Build) nightly (x86_64)
-- | --
Revision | 62eb262cdaae9ef819aadd778193781455ec7a49-refs/branch-heads/4389@{#1534}
OS | macOS Version 11.2.3 (Build 20D91)

## Version/Channel Information:

- Can you reproduce this issue with the current release? no
- Can you reproduce this issue with the beta channel? yes
- Can you reproduce this issue with the nightly channel? yes

cc: @karenkliu @rebron @simonhong @bsclifton

**Comment by simonhong** [comment]
Hmm, it seems regression. `1.23.x` didn't have this issue.

**Comment by bsclifton** [comment]
It's possible this regressed with https://github.com/brave/brave-core/pull/8120
cc: @petemill

**Comment by GeetaSarvadnya** [comment]
1.21.x - Not Reproducible
1.22.x - Not Reproducible
1.23.x - Reproducible
1.24.x - Reproducible

**Comment by simonhong** [comment]
My local build is `1.23.33` and that version doesn't have this issue.

**Comment by GeetaSarvadnya** [comment]
I have verified in `1.23.43` and the issue is reproducible in this version
![Top tiles_1 23 43](https://user-images.githubusercontent.com/38657976/111604889-ccf3c180-87fb-11eb-8b6a-31c4ab5d3cdd.gif)

**Comment by bsclifton** [comment]
This doesn't seem to be a problem anymore w/ Chromium 90; @simonhong can you verify? Should we still fix this? 1.23.x will be first release w/ Chromium 90 and is also first build with this code; so I don't think we need a fix

**Comment by bsclifton** [comment]
OK confirmed it still IS an issue; you just need to use TAB key to focus 👍

**Comment by stephendonner** [comment]
Verified `FIXED` using 

Brave | 1.24.26 Chromium: 90.0.4430.30 (Official Build) nightly (x86_64)
-- | --
Revision | 5674335ff855e43f3bccf8cfc29a779bdf0d067f-refs/branch-heads/4430@{#532}
OS | macOS Version 11.2.3 (Build 20D91)

The focus ring is now back to spec.

<img width="1468" alt="Screen Shot 2021-03-25 at 9 27 51 AM" src="https://user-images.githubusercontent.com/387249/112508162-8991d980-8d4c-11eb-897e-e980d39a5fcf.png">
<img width="181" alt="Screen Shot 2021-03-25 at 9 28 00 AM" src="https://user-images.githubusercontent.com/387249/112508173-8d256080-8d4c-11eb-8972-99ee7add760b.png">
